lib/housekeeping.js

```diff
@@ -123,9 +124,13 @@ class HouseKeeper {
         assert(active.hasOwnProperty('count'));
         assert(unused.hasOwnProperty('gb'));
         assert(unused.hasOwnProperty('count'));
-
+        
```

Comment:
trailing space?

Reply:
Probably but I wasn't sure about the whitespace. I'll look more into this in a few days.

I'm blaming Atom for the trailing space. It keeps creating random indents. There's probably a setting in Atom that I could turn off.

Reply:
https://en.wikipedia.org/wiki/Vim_(text_editor)